modules/base.nix

```diff
@@ -116,5 +116,9 @@
   programs.fish.enable = true;
   programs.zsh.enable = true;
 
+  # Raise UDP send/recv buffer size since we rely _very_ heavily on QUIC/WireGuard
+  boot.kernel.sysctl."net.core.wmem_max" = 7500000;
+  boot.kernel.sysctl."net.core.rmem_max" = 7500000;
```

Comment:
This could maybe potentially perhaps bite us. Once we confirm the fix is working we should experiment with lower values IMO